package.json

```diff
@@ -36,6 +36,8 @@
     "test": "npx jest",
     "test:integration": "./scripts/test-integration",
     "test:watch": "jest --watch",
+    "test-storybook": "test-storybook",
+    "test-storybook:watch": "test-storybook --watch",
```

Comment:
I don't think we need this one actually. You can run this locally by doing `npm run test-storybook -- --watch`